lib/classes/news.dart

```diff
@@ -0,0 +1,47 @@
+class News {
+  const News({
+    required this.by,
```

Comment:
Should this only have the factory instantiate it? If so, all of these can be private.